endurain/docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: endurain_app_1
+      APP_PORT: 8080
+      PROXY_AUTH_ADD: "false"
+
+  app:
+    image: ghcr.io/joaovitoriasilva/endurain:v0.11.2@sha256:2bd4c73e564a152ac17e336c7923e36cc6a28a3be6e67664db7ba11659455033
+    # not working rootless right now
+    # user: "1000:1000"
+    environment:
+      DB_TYPE: postgres
+      DB_HOST: endurain_db_1
+      DB_PORT: 5432
+      DB_PASSWORD: endurain
+      SECRET_KEY: ${APP_PASSWORD}
+      ENDURAIN_HOST: http://${DEVICE_DOMAIN_NAME}:8865
+      BEHIND_PROXY: "true"
+      FERNET_KEY: changeme
+      GEOCODES_MAPS_API: changeme
```

Comment:
Should we include a .env for these do you think? Or is the usecase not high enough?

Reply:
Added a .env file, the `FERNET_KEY` can be generated on device.